src/posts/how-to-use-JAWS-on-Windows

```diff
@@ -0,0 +1,47 @@
+---
+title: HOWTO: Use JAWS on Windows
+description: "Step-by-step guide on effectively using JAWS, a screen reader software, to navigate Windows, read documents, browse the web, and explore advanced features for individuals with visual impairments."
+category: How-to
+author: Ilknur Eren
+date: 2022-06-04
+tags:
+  - howto
+---
+
+
+## Introduction
+JAWS, Job Access With Speech, is a screen reader for Windows operating system users. JAWS assists individuals with visual impairments with interacting with computers by helping them navigate applications, read documents and browse the web on Windows operating systems. 
+
+In this article, we will provide step-by-step instructions on how to start using JAWS on Windows.
+
+## Install JAWS
+First, the user needs to install JAWS in order to use it. JAWS have several subscription plans for their users. JAWS can be installed on their website, [Freedom Scientific website](https://www.freedomscientific.com/products/software/jaws/). You can purchase a valid license that fits you. After purchasing a license, you can find JAWS on your download file or a file you saved the program in. Double click on it to start the installation process. The installer will guide you through the installation process, which will help you choose your desired language and installation options. After successfully downloading JAWS, restart your computer to finalize the set-up. You now have JAWS and are ready to use it.
```

Comment:
Does JAWS still offer free limited-use VMs for testing?

Reply:
@davatron5000 JAWS has a 40 minute demo mode for Windows.

There is also Assitiv Labs that will provide remote screen reader testing for all devices but requires subscription.
https://assistivlabs.com/